snmp/snmp-device-gateway/src/main/java/com/cumulocity/agent/snmp/persistence/AbstractQueue.java

```diff
@@ -74,13 +81,6 @@ public AbstractQueue(String queueName, File persistenceFolder) {
 
         log.info("Creating/Loading '{}' Queue, backed by the folder '{}'", this.name, persistenceFolder.getPath());
 
-        // Since the rolling cycle is set to minutely and we also delete the old files periodically,
-        // we are fine with not shrinking the data files.
-        // Hence we disable the data file shrinking, unless it is explicitly requested by setting the
-        // system property "chronicle.queue.disableFileShrinking"
-        QueueFileShrinkManager.DISABLE_QUEUE_FILE_SHRINKING = Boolean.parseBoolean(
```

Comment:
`DISABLE_QUEUE_FILE_SHRINKING` is now final and not possible to assign value to it